packages/calcite-components/src/components/stepper-item/stepper-item.scss

```diff
@@ -120,74 +153,79 @@
 :host .stepper-item-description {
   @apply flex w-full;
 }
+
 :host .stepper-item-heading {
-  @apply text-color-2 mb-1 font-medium;
+  @apply mb-1;
+
+  color: var(
+    --calcite-stepper-item-heading-text-color,
+    var(--calcite-internal-stepper-item-heading-text-color, var(--calcite-color-text-2))
+  );
+  font-weight: var(--calcite-font-weight-medium);
 }
+
 :host .stepper-item-description {
-  @apply text-color-3;
+  color: var(
+    --calcite-stepper-item-description-text-color,
+    var(--calcite-internal-stepper-item-description-text-color, var(--calcite-color-text-3))
```

Comment:
Given that --calcite-stepper-item-description-text-color, --calcite-stepper-item-icon-color, and --calcite-stepper-item-number-text-color all share the same default value. Let's set them to all fallback to --calcite-stepper-item-text-color

`color: var(
    --calcite-stepper-item-description-text-color,
    var(--calcite-stepper-item-text-color, var(--calcite-internal-stepper-item-description-text-color, var(--calcite-color-text-3))));`

Reply:
Curious if there is an advantage of setting to `--calcite-stepper-item-text-color` as fallback color  and then adding `--calcite-internal-stepper-item-description-text-color` as fallback for `--calcite-stepper-item-text-color . 


This can be really confusing in terms of readability me thinks and having internal tokens for overriding tokens based on states would be much clearer. 

Looking at  `--calcite-stepper-item-text-color`  much closer and realized it isn't being used any where. We can remove it. Thoughts?